src/components/atoms/MainSliderCard/mainSliderCard.m.css

```diff
@@ -0,0 +1,88 @@
+.base {
+  --background-color: var(--white);
+  --accent-color: var(--yellowish-orange);
+  background-color: var(--background-color);
+}
+.base :global .contributor {
+  margin-bottom: var(--space1);
+}
+
+.base :global .image-wrapper {
+  /* @include image-aspect-ratio(5, 3); */
+  position: relative;
+  padding-top: 60%;
+}
+.base :global .image-wrapper:before {
+  --background-color: var(--black);
+  z-index: 1;
+  display: block;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  content: "";
+  height: 70px;
+  opacity: 0.5;
+  background-image: linear-gradient(to bottom, transparent, var(--black));
+}
```

Comment:
same here as well, you can nest this inside base

Reply:
done